File: app/src/main/java/com/softartdev/notecrypt/model/Note.java
package com.softartdev.notecrypt.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class Note extends RealmObject {
    public static final String NOTE_ID = "note_id";

    @PrimaryKey
    private long id;

    private String title;
    private String text;

    public long getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }
}

File: app/src/main/java/com/softartdev/notecrypt/model/Note.kt
package com.softartdev.notecrypt.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import java.util.*

open class Note : RealmObject() {
    @PrimaryKey var id: Long = 0
    var title: String = ""
    var text: String = ""
    var dateCreated = Date()
    var dateModified = Date()
}
